misc/python/materialize/bintray.py

```diff
@@ -0,0 +1,146 @@
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+"""A Bintray API client.
+
+Consult the [official Bintray API documentation][official-docs] for details.
+Only the operations that are presently necessary are implemented.
+
+[official-docs]: https://bintray.com/docs/api/
+"""
+
+from typing import IO, Sequence
+from typing_extensions import Final
+from requests import Session, Response
+import requests
+
+API_BASE: Final = "https://api.bintray.com"
+
+HTTPError = requests.HTTPError
```

Comment:
Is there a reason this isn't part of `from requests import ..`?

Reply:
Oops, dead code. Thanks. Really miss Rust's lint on that.